src/lucky/page_helpers/html_helpers.cr

```diff
@@ -0,0 +1,159 @@
+# These helper methods will write directly to the view.
+module Lucky::HTMLHelpers
```

Comment:
What do you think about HTMLTextHelper? So it’s clear it’s for text?

Reply:
lol. I originally had that, and then second guessed myself. I'll go back to that.